src/Main.ts

```diff
@@ -56,7 +54,7 @@ class Main {
      * @param params the options array to get the item from
      * @param index if provided gets the corresponding item from the array
      */
-    public loadFromArray(tagId: string, params: RecursivePartial<IOptions>[], index?: number): Container | undefined {
+    public async loadFromArray(tagId: string, params: RecursivePartial<IOptions>[], index?: number): Promise<Container | undefined> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)